src/components/gauge/gauge.e2e.ts

```diff
@@ -0,0 +1,213 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+describe("gx-gauge", () => {
+  let element: E2EElement;
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+  });
+
+  it("should be drawn without attributes attrs", async () => {
+    await page.setContent("<gx-gauge></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    expect(element).toBeTruthy();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+
+  /////////////////// Type test ///////////////////////
+  it("should render with the line type", async () => {
+    await page.setContent("<gx-gauge type='line'></gx-gauge>");
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  });
+
+  it("should render with the circle type", async () => {
+    await page.setContent(`
+      <gx-gauge type="circle"></gx-gauge>
+    `);
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    await page.waitForChanges();
+    const svg = await element.find("svg");
+    const svgCircle = await element.find("svg circle");
+    await page.waitForChanges();
+    expect(svg).toBeTruthy();
+    expect(svgCircle).toBeTruthy();
+  });
+
+  /////////////////// Values test ///////////////////////
+  it("should automatically set the default minimum value", async () => {
+    await page.setContent(`
+      <gx-gauge show-value="true"></gx-gauge>
+    `);
+    await page.waitForChanges();
+    element = await page.find("gx-gauge");
+    const minValueDisplayer = await element.find("span.minValue");
+    expect(await element.getProperty("minValue")).toEqual(0);
+    expect(minValueDisplayer.innerHTML).toEqual("0<span></span>");
```

Comment:
I think using `textContent` would simplify this case and allow you to remove `<span></span>` from the expected value.